app/assets/stylesheets/image_x.scss

```diff
@@ -147,6 +147,39 @@ $image-thumb-slider-background-color: $sul-background;
   text-align: center;
 }
 
+.#{$namespace}-image-x-restricted-thumb-container {
+  .#{$namespace}-image-x-auth-link {
+    @include rounded();
+    background: rgba($sul-background, 0.65);
+    bottom: 10px;
+    left: 10px;
+    padding: 5px;
+    position: absolute;
+  }
+
+  a {
+    @include stanford-only;
+  }
+}
+
+.#{$namespace}-image-x-iiif-drag-and-drop-link {
+  border-bottom: none !important;
```

Comment:
In this case, I think I can live with letting it slide.  Typically, yes, we should care about it as it indicates that the specificity in the css is too complicated, but the amount of change that would require at this point would be to high to consider it viable.